Netable/Netable/Netable.swift

```diff
@@ -100,10 +100,11 @@ open class Netable {
         let requestInfo = LogEvent.RequestInfo(
             urlString:  urlRequest.url?.absoluteString ?? "UNDEFINED",
             method: request.method,
-            headers: urlRequest.allHTTPHeaderFields ?? [:],
-            params: try? request.parameters.toParameterDictionary(encodingStrategy: request.jsonKeyEncodingStrategy))
+            headers: urlRequest.allHTTPHeaderFields ?? [:]
+        )
 
         log(.requestStarted(request: requestInfo))
+        log(.requestBody(body: request.safeParameters))
```

Comment:
One potential issue with doing the redaction by default inside Netable like this, it means that it is impossible for a user to turn off redaction globally in any way. You'd have to go through an add a full set of safe properties to all requests. 

I'd say we should at at least one (maybe both) of:
 - A way to just turn this off entirely in the configuration.
 - A way to bypass this inside a custom logger.

Reply:
Yeah I kind of went back and forth on this as well, as a starter do you think it makes sense to just _not_ redact logs by default?

Reply:
It might. Steamclog can be a little opinionated about that because it's intended for internal use so we can just play to OUR defaults, but I suspect that most users wouldn't care, and so probably the right thing is to is just have it as an option. That'd also make it more logical to name the config parameter `enableLogRedaction`, which might be a good thing anyway, `disableX = true` style things always make my brain stumble over the double negative a bit.